ethereum_history_api/circuits/lib/src/verifiers/transaction_test.nr

```diff
@@ -60,3 +60,60 @@ mod assert_tx_proof {
         assert_tx_proof(proof_invalid_depth, transactions_root);
     }
 }
+
+mod assert_tx_equals {
+    use crate::verifiers::transaction::assert_tx_equals;
+    use crate::fixtures::mainnet::{
+        cancun::small_block::transaction::{transaction, encoded_tx, tx_type},
+        homestead::fork::transaction::{transaction as legacy_transaction, encoded_tx as legacy_encoded_tx, tx_type as legacy_tx_type}
+    };
+
+    #[test]
+    fn success() {
+        assert_tx_equals(tx_type, encoded_tx, transaction);
+    }
+
+    #[test]
+    fn legacy_tx() {
```

Comment:
Maybe: success_legacy_tx() ?

Reply:
What's the benefit of adding this prefix here?
When we run tests from the CLI and this one fails - it's clear from error message that it was supposed to succeed.
When we are reading the code - it's clear from the #[test] macro

Reply:
I think it would explain what is the difference between the first and second test.  Or maybe the first one instead of "success" could be named to show that it's not legacy. Just to show the difference clearly